python/delta_sharing/tests/test_rest_client.py

```diff
@@ -378,3 +379,13 @@ def test_list_files_in_table_partitioned_different_schemas(
             ),
         ),
     ]
+
+
+def test_list_table_changes(
+    rest_client: DataSharingRestClient,
+):
+    try:
+        rest_client.list_table_changes(Table(name="x", share="y", schema="z"), CdfOptions())
+        assert False
+    except Exception as e:
```

Comment:
This is to test the 'not implemented' error?

Reply:
Yes.
I'm implementing this in the next PR.